src/client/datascience/jupyter/kernels/kernelDependencyService.ts

```diff
@@ -71,6 +71,8 @@ export class KernelDependencyService implements IKernelDependencyService {
             ]);
             if (response === InstallerResponse.Installed) {
                 return KernelInterpreterDependencyResponse.ok;
+            } else if (response === InstallerResponse.Ignore) {
```

Comment:
From the code it looks like ignoring or closing the prompt also logs this. In that case I would expect a KernelInterpreterDependencyResponse.cancel not .failed. Can we distinguish the two?

Reply:
I guess we can't any non-ok response the same, so maybe ok as is.

Reply:
I don't believe it returns that value when canceling. It returns InstallerResponse.Disabled. Yeah verified that with the debugger. It only returns that when it fails.

Reply:
Ok, that's odd. I believe the debugger. Just in line 164 in productInstaller.ts it looks like a 'No' response from the prompt would lead to disable.

Reply:
It doesn't go thorugh the productInstaller's prompt. The kernel dependency stuff uses its own.